roles/etcd/tasks/gen_certs_script.yml

```diff
@@ -114,7 +118,9 @@
 - name: Gen_certs | Prepare tempfile for unpacking certs                        
   shell: mktemp /tmp/certsXXXXX.tar.gz                                          
   register: cert_tempfile                                                       
-                                                                                
```

Comment:
It doesn't work that way... you have to loop with multiple delegate_to targets (which is slower). And we use the same rule many times.